js/ui/sortable.js

```diff
@@ -0,0 +1,206 @@
+import $ from "../core/renderer";
+import registerComponent from "../core/component_registrator";
+import { extend } from "../core/utils/extend";
+import Draggable from "./draggable";
+import { getPublicElement } from "../core/utils/dom";
+
+var SORTABLE = "dxSortable",
+    DEFAULT_ITEMS = "> *",
+
+    SOURCE_CLASS = "source",
+    PLACEHOLDER_CLASS = "placeholder";
+
+var Sortable = Draggable.inherit({
+    _getDefaultOptions: function() {
+        return extend(this.callBase(), {
+            orientation: "vertical",
+            clone: true
+        });
+    },
+
+    _init: function() {
+        this.callBase();
+    },
+
+    _getItemsSelector: function() {
+        return this.callBase() || DEFAULT_ITEMS;
+    },
+
+    _removePlaceholderElement: function() {
+        if(this._$placeholderElement && !this._$placeholderElement.hasClass(this._addWidgetPrefix(SOURCE_CLASS))) {
+            this._$placeholderElement.remove();
```

Comment:
variable _$placeholderElement should be cleared:
this._$placeholderElement = null;